app/assets/javascripts/student_profile_v2/math_details.js

```diff
@@ -1,12 +1,132 @@
 (function() {
-  window.shared || (window.shared = {});
-  var dom = window.shared.ReactHelpers.dom;
-  var createEl = window.shared.ReactHelpers.createEl;
-  var merge = window.shared.ReactHelpers.merge;
+window.shared || (window.shared = {});
+var dom = window.shared.ReactHelpers.dom;
+var createEl = window.shared.ReactHelpers.createEl;
+var merge = window.shared.ReactHelpers.merge;
+
+var ProfileChartSettings = window.ProfileChartSettings;
+var HighchartsWrapper = window.shared.HighchartsWrapper;
+
+var MathDetails = window.shared.MathDetails = React.createClass({
```

Comment:
@erose Can you fixup the indentation here so that these initial lines within the function are indented?